frontend/src/scenes/trends/ShownAsFilter.js

```diff
@@ -23,8 +23,8 @@ export function ShownAsFilter({ shown_as, onChange }) {
                     <small>
                         <i>
                             Stickiness shows you how many days users performed an action within the timeframe. If a user
-                            did an action on Monday and came back and did it twice on Friday, it would be listed as "2
-                            days" in the chart.
+                            performed an action on Monday and came back and did it twice on Friday, it would be listed
```

Comment:
One more nit: they don't need to perform an action twice on friday right just once monday and once friday

Reply:
Ah this was to indicate that even if they did the action multiple times on the same day, it would still only could as 2 days. Maybe this is obvious and confusing in this way though?